### DISCUSSION TITLE
Azure DevOps integration

### DISCUSSION THREAD

**Discussion by avivapinchas** [ParabolInc/parabol#5459] [New features]
Approx. 15 requests here. 

User requests: #3004

**Comment by enriquesanchez** [comment]
Azure DevOps was one of the top most requested integrations from the [Parabol Customer Feedback Survey](https://parabol.slack.com/archives/C08FL5UE7/p1633422567280700) 🔒.

**Comment by jordanh** [comment]
Plus one from [this user](https://app.hubspot.com/contacts/3888472/contact/10202251)

**Comment by drewhousman1010** [comment]
Plus one from [this user](https://app.hubspot.com/contacts/3888472/contact/3538751) 🔒 

> We would also love to use Parabol for estimating selected user stories from Azure Devops.
> Are you planning an integration with Azure DevOps?

**Comment by jordanh** [comment]
We've implemented this now. It will be live soon.
